src/ApiPort/App.config

```diff
@@ -1,20 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
-      <dependentAssembly>
-        <assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.0.1.0" newVersion="4.0.1.0" />
-      </dependentAssembly>
-    </assemblyBinding>
-  </runtime>
-  <!-- Uncomment this to see diagnostics -->
-  <!--<system.diagnostics>
```

Comment:
Is it possible to keep this? It is nice to have the trace listener when I need a stack trace for users who have failures in their client application since we usually abstract it into a "Failed" output. :/

Reply:
No problem - I've added it back